### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by g-bougard** [glpi-project/glpi-agentmonitor#2] [Announcements]
Thank you @redddcyclone to have shared this project with GLPI community !

**Comment by redddcyclone** [comment]
I'm glad I could contribute to the community! I hope this will be useful to everyone 😄

**Reply by g-bougard** [reply]
Just for you to know, I added a Contributions section in our public documentation, see: https://glpi-agent.readthedocs.io/en/latest/installation/index.html#contributions
I also published again the 1.0 and it seems VT don't complain anymore ;-)
See: https://github.com/glpi-project/glpi-agentmonitor/releases/tag/1.0.0

**Reply by redddcyclone** [reply]
This is good! If the false positives stop coming ever again, would there be a possibility to integrate it into the Windows agent package in the future?

**Reply by g-bougard** [reply]
I was thinking also to finally integrate it in the MSI packaging as it still would be more appropriate to install or uninstall it as the same time.
Anyway, I'm still scared by DeepInstinct false positive which was always triggered during my last build tests.
I suspect we was lucky yesterday with the signed build I published. Maybe this one doesn't contain a footprint DeepInstinct recognizes as malicious.
To conclude, the currently published could still be included in the package.

**Reply by redddcyclone** [reply]
That's strange. Maybe the built binaries are detected over time by the antiviruses as not really being malicious? I mean, there was no change in the code between the last builds reported as malicious and this one, was there?

Maybe in the future the false positives related to this project will disappear for good, but only time will tell.

**Reply by g-bougard** [reply]
Code didn't change but the binary changed as it is signed and the timestamp at which time it is signed changes. So the whole file footprint changes too.

Now only DeepInstinct check is in trouble and only with the 32 bits version. I really hope this would be fixed soon or later.
